build.gradle

```diff
@@ -315,7 +316,7 @@ dependencies {
 dependencyCheck {
     // Specifies if the build should be failed if a CVSS score above a specified level is identified.
     // range of 0-10 fails the build, anything greater and it doesn't fail the build
-    failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'true' ? 0 : 11
+    failBuildOnCVSS = System.getProperty('dependencyCheck.failBuild') == 'false' ? 0 : 11
```

Comment:
Is there a reason this is no longer set to true? I thought we required this to inform us of vulnerable dependencies?